Source/Bogus/Bogus.csproj

```diff
@@ -5,7 +5,7 @@
     </PackageReleaseNotes>
     <Version>0.0.0-localbuild</Version>
     <Authors>Brian Chavez</Authors>
-    <TargetFrameworks>net40;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net40;netstandard1.3;netstandard2.0;net60</TargetFrameworks>
```

Comment:
@prasadtelkikar , the proper TFM for .NET 6 is `net6.0` (with period), as similarly shown [here](https://github.com/dotnet/wcf/blob/4d7e90ec682af0a23fe0bafe0a51d891abcb4ccc/src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.Ref.csproj#L5) in Microsoft's usage.

Additionally, the documentation for .NET TFMs is here:
* https://docs.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-frameworks

![image](https://user-images.githubusercontent.com/478118/141890583-24a3a527-9964-4f04-80b0-760803ed5f78.png)

Reply:
I fixed it

Reply:
Hi @bchavez, I fixed it